src/trackers/COMMON.py

```
@@ -853,7 +854,7 @@ def format_bbcode(self, parsed_mediainfo):
             bbcode_output += "\n"
             return bbcode_output
 
-    async def get_bdmv_mediainfo(self, meta, remove=None, char_limit=None):
+    async def get_bdmv_mediainfo(self, meta: dict[str, Any], remove: list[str] = [], char_limit: int = 0):
```

Ruff (0.14.10)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)